runner.go

```diff
@@ -143,13 +143,26 @@ func PassThrough(types ...Type) Runner {
 	if len(types) == 0 {
 		return passThroughSingleton
 	}
-	return &passThrough{types}
+	return &passThrough{types, StringsSet{}}
+}
+
+// PassThroughWithScopes returns a runner that lets all of its input through as-is
+func PassThroughWithScopes(scopes StringsSet, types ...Type) Runner {
+	if len(types) == 0 {
+		return passThroughSingleton
```

Comment:
using singleton will discard provided scopes. what do you think about:
```suggestion
		panic("scopes without types are not allowed")
```